src/publish/mod.rs

```diff
@@ -31,25 +31,48 @@ impl CaPublisher {
     }
 
     pub async fn publish(&self, ca_handle: &Handle, actor: &Actor) -> Result<(), Error> {
-        let ca = self.caserver.get_ca(ca_handle).await?;
+        // Since this is called by the scheduler, this acts as a no-op for new CAs which do not yet have any repository configured.
```

Comment:
Does this comment still make sense? I can see in the old code what it referred to but it's not clear to me now.